.github/workflows/run_tests.yml

```diff
@@ -71,7 +71,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Migrate Database
-        uses: joshuaavalon/flyway-action@v1
+        uses: joshuaavalon/flyway-action@v3.0.0 # flyway-action 3.0.0 uses flyway v9, which should (if we do our jobs) match what's in Dockerfile.db_clean and Dockerfile.db_migrations
```

Comment:
Same comment. idk if we want to lock this to a minor and major version unless we need to

Reply:
The only thing I noted is that I think GitHub actions don't actually _know_ what SemVer is, they just work on tags. The only reason `v1` worked before is there's a `v1` tag on that repo. Sadly, they stopped doing that on that action and now only have `v3.0.0`

https://github.com/joshuaavalon/flyway-action/tags
https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management

GitHub recommends making multiple tags for each version, and including a "major only" tag that moves as updates get pushed, but seems like this action doesn't follow that.

Reply:
Cool cool TIL. Didn't realize that action was tagged like that 👍